src/snowflake/connector/config_manager.py

```diff
@@ -333,7 +334,14 @@ def read_config(
                     and filep.stat().st_uid != os.getuid()
                 )
             ):
-                warn(f"Bad owner or permissions on {str(filep)}")
+                # for non-Windows, suggest change to 0600 permissions.
+                chmod_message = (
+                    f". To change owner, run chown $USER {str(filep)}. To restrict permissions, run chmod 0600 {str(filep)}."
+                    if platform.system() != "Windows"
```

Comment:
Could you please use `IS_WINDOWS`? Like we do in the tests

Reply:
didn't know this existed in the main source code, great! Thought it was a test flag.